app/PaymentType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PaymentType extends Model
+{
+    protected $fillable = ['name'];
+
+    const INCOME = 'INCOME';
+    const EXPENSE = 'EXPENSE';
+    const KKT = 'KKT';
+    const NETREG = 'NETREG';
+    const TYPES = [
+        self::INCOME,
+        self::EXPENSE,
+        self::KKT,
+        self::NETREG,
+    ];
```

Comment:
I think later more Payment types could be added dinamically.
These will be meaningful if transactions with the same type would be summarized in the statistics, while we could add these transactions separately (eg. T-shirts, routers besides kkt/netreg). Income and Expense would mean other, unique transactions that should be displayed in the statistics separately.
I don`t think constans would be a solution for this.

Reply:
I see, thought that this would not be a dynamic class. Let's keep these for now and remove when we will move to dynamic payment types.

Reply:
okay